lib/xcode/install.rb

```diff
@@ -103,6 +103,10 @@ def install_dmg(dmgPath, suffix = '', switch = true, clean = true)
       end
 
       FileUtils.rm_f(dmgPath) if clean
+
+      return if version.nil?
+      xcode = seedlist.find { |x| x.name == version }
+      `open #{xcode.release_notes_url}` unless xcode.nil? || xcode.release_notes_url.nil?
```

Comment:
I think it would be better to move this up to `install_version` (https://github.com/neonichu/xcode-install/blob/master/lib/xcode/install.rb#L113) as `install_dmg` is already crowded and has a lot of parameters.